src/main/java/duke/parts/Ui.java

```diff
@@ -0,0 +1,68 @@
+package duke.parts;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    static String LINE = "    ____________________________________________________________";
+    static String INDENT = "    ";
+    Scanner sc = new Scanner(System.in);
```

Comment:
May specify all as `private`? I will omit other similar problems from this point.